cmd/eskip/media_defaults.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+type defaultFunc func(in, out *medium) (input, output *medium, err error)
+
+// map command string to defaults
+var commandToDefaultMediums = map[command]defaultFunc{
+	check:  defaultRead,
+	print:  defaultRead,
+	upsert: defaultWrite,
+	reset:  defaultWrite,
+	delete: defaultWrite}
+
+func defaultRead(in, out *medium) (input, output *medium, err error) {
+	input = in
+	if input == nil {
+		// take innkeeper as default
+		input, err = processEtcdArgs(defaultEtcdUrls, defaultEtcdPrefix)
+
+		if err != nil {
+			return nil, nil, err
+		}
+	}
+	return input, nil, nil
```

Comment:
when using named return args, it is nicer to explicitly set the value for each of them, and just simply call 'return' without args